crates/uv-resolver/src/pubgrub/dependencies.rs

```diff
@@ -21,22 +21,16 @@ impl PubGrubDependencies {
         requirements: &[Requirement],
         constraints: &[Requirement],
         overrides: &Overrides,
-        extra: Option<&ExtraName>,
-        source: Option<&PackageName>,
+        source_name: Option<&PackageName>,
+        source_extra: Option<&ExtraName>,
```

Comment:
I just renamed and re-ordered these (especially since `extra` got shadowed below).